modules/apps/portal-template/portal-template-react-renderer-impl/src/main/java/com/liferay/portal/template/react/renderer/internal/ReactRendererUtil.java

```diff
@@ -40,6 +45,8 @@ public class ReactRendererUtil {
 
 	public static void renderReact(
 			ComponentDescriptor componentDescriptor, Map<String, Object> props,
+			AbsolutePortalURLBuilderFactory absolutePortalURLBuilder,
+			DeferredScriptsManager deferredScriptsManager,
```

Comment:
I think bchan will want these params sorted alphabetically.

Reply:
Yeah, maybe I can refactor that and reorder them